python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -92,105 +117,202 @@ def optimize(self, model,
                     try:
                         if accelerator is None and use_ipex is False:
                             accelerated_model = model
                         else:
-                            # TODO: remove the logging of tracing
-                            accelerated_model = Trainer.trace(model=model,
-                                                              accelerator=accelerator,
-                                                              use_ipex=use_ipex,
-                                                              input_sample=input_sample)
-                        # TODO: 100 trial run is now fixed, we may make it adjusted intelligently.
-                        result_map[method] = {}
-
-                        def func_test(model, input_sample):
-                            model(*input_sample)
-                        result_map[method]["latency"] =\
-                            _throughput_calculate_helper(100, func_test,
-                                                         accelerated_model, input_sample)
-                        if validation_data is not None and metric is not None:
-                            result_map[method]["accuracy"] =\
-                                _accuracy_calculate_helper(accelerated_model,
-                                                           metric, validation_data)
-                        result_map[method]["model"] = accelerated_model
+                            if accelerator in ("jit", None):
+                                accelerated_model = Trainer.trace(model=model,
+                                                                  accelerator=accelerator,
+                                                                  use_ipex=use_ipex,
+                                                                  channels_last=use_channels_last,
+                                                                  input_sample=input_sample)
+                            else:
+                                accelerated_model = Trainer.trace(model=model,
+                                                                  accelerator=accelerator,
+                                                                  input_sample=input_sample,
+                                                                  # remove output of openvino
+                                                                  logging=False)
                     except Exception as e:
                         print(e)
+                        continue
 
                 # if precision is int8 or bf16, then we will use quantize method
-                if split_method[1] == "int8" or split_method[1] == "bf16":
-                    precision = split_method[1]
-                    q_method = None if len(split_method) < 4 else split_method[3]
+                elif precision in ("int8", "bf16"):
+                    ort_method = _detect_ort_method(method)
```

Comment:
`_detect_quantization_method`? since this output will be assigned to `method` in `Trainer.quantize`

Reply:
done